internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,17 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	examplesJSON, err := prettyJSON([]byte(sb.String()))
```

Comment:
Oh and another nit for something we could improve as well.

We have been using a `strings.Builder` because we never needed the underlying bytes directly. Now that we need to use a JSON `[]byte` slice for `json.Indent`, we could:
1. Refactor `sb := &strings.Builder{}` to `buf := &bytes.Buffer{}`
2. Change `sb.WriteString` to `buf.WriteString` everywhere
3. Update this line to call `prettyJSON(sb.Bytes())`

```go
buf := &bytes.Buffer{}
buf.WriteString(`[`)
for i, example := range u.crs {
	buf.WriteString(example)
	if i < len(u.crs)-1 {
		buf.WriteString(`,`)
	}
}
buf.WriteString(`]`)
examplesJSON, err := prettyJSON(buf.Bytes())
if err != nil {
	return err
}
```

Reply:
God spot. Tks.